PRChecker/Shared/Models/PullRequest.swift

```diff
@@ -7,21 +7,99 @@
 
 import Foundation
 import Apollo
+import SwiftUI
```

Comment:
Just a thought, it might be better to implement `var color: Color` in view/viewModel layer rather than importing `SwiftUI` into model layer

Reply:
Good call... any ideas on where to put it?

Reply:
Actually... we use PullRequest as a view model right now. It might just need a name change to reflect that

Reply:
I'm currently working on creating `PRItemType` enum, maybe we could move to there later. Let skip it for now!